composer.json

```diff
@@ -2,10 +2,10 @@
     "name": "derhasi/boxfile",
     "description": "Command to handle freistil Boxfiles anywhere",
     "require": {
-        "symfony/yaml": "2.* || ~3.2.0",
+        "symfony/yaml": "2.* || 3.*",
         "symfony/console": "2.* || 3.*",
-        "symfony/config": "~2.6 || ~3.2.0",
-        "derhasi/symlinker": ">=0.2.2"
+        "symfony/config": "~2.6 || 3.*",
+        "derhasi/symlinker": ">0.4.0"
```

Comment:
@hctom this change is unnecessary. `>=0.2.2` also allows everything that is `>0.4.0`.

Reply:
@derhasi Yes you are right. I reverted that commit in #5